interface/forms/eye_mag/php/eye_mag_functions.php

```
@@ -2086,14 +2087,14 @@ function build_PMSFH($pid)
 function display_PMSFH($rows, $view = "pending", $min_height = "min-height:344px;")
 {
     global $PMSFH;
-    global $pid;
+    $pid = PatientSessionUtil::getPid();
     global $PMSFH_titles;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$PMSFH_titles'. (undefined)

(UnusedLocalVariable)